conf/vertebrates/production_reg_conf.pl

```diff
@@ -51,8 +51,8 @@
 my @metazoa_overlap_species = qw(drosophila_melanogaster caenorhabditis_elegans);
 Bio::EnsEMBL::Compara::Utils::Registry::remove_species(\@metazoa_overlap_species);
 my $metazoa_overlap_cores = {
-    'drosophila_melanogaster' => [ 'mysql-ens-vertannot-staging', "drosophila_melanogaster_core_104_9" ],
-    'caenorhabditis_elegans'  => [ 'mysql-ens-vertannot-staging', "caenorhabditis_elegans_core_104_269" ],
+    'drosophila_melanogaster' => [ 'mysql-ens-vertannot-staging', "drosophila_melanogaster_core_105_9" ],
+    'caenorhabditis_elegans'  => [ 'mysql-ens-vertannot-staging', "caenorhabditis_elegans_core_105_269" ],
```

Comment:
```suggestion
    'drosophila_melanogaster' => [ 'mysql-ens-vertannot-staging', "drosophila_melanogaster_core_$curr_release_9" ],
    'caenorhabditis_elegans'  => [ 'mysql-ens-vertannot-staging', "caenorhabditis_elegans_core_$curr_release_269" ],
```
I realise you simply changed `104` to `105` following suit, but if these remain stable, then nothing will need to be changed next release :crossed_fingers:

Reply:
Yes, but I first checked
`mysql-ens-vertannot-staging -Ne "SHOW DATABASES LIKE 'drosophila_melanogaster_core%'"`
`mysql-ens-vertannot-staging -Ne "SHOW DATABASES LIKE 'caenorhabditis_elegans_core%'"`